zilliqa/src/consensus.rs

```diff
@@ -739,6 +739,13 @@ impl Consensus {
         content
     }
 
+    pub fn transaction_count_for_account(&self, account: Address) -> u64 {
+        let current_nonce = self.state.must_get_account(account).nonce;
+
+        self.transaction_pool
+            .transaction_count(account, current_nonce)
```

Comment:
Bit of a nit, but I'd prefer if all these methods were called `pending_transaction_count`. If I see the name `transaction_count` my first guess would be that they returned the number of mined transactions from this account, which is specifically what these implementations don't do.